modules/installation-configure-proxy.adoc

```diff
@@ -94,6 +103,22 @@ Only the Proxy object named `cluster` is supported, and no additional
 proxies can be created.
 ====
 
+ifdef::aws-upi,aws-restricted-upi,gcp-upi[]
+
+When the cluster-wide proxy is enabled, you must manually create the Ingress DNS records.
+
+// TODO: xref: https://docs.openshift.com/container-platform/4.2/installing/installing_aws_user_infra/installing-aws-user-infra.html#installation-create-ingress-dns-records_installing-aws-user-infra
```

Comment:
If the TODO stays, please add some context for others to understand.

Reply:
Unfortunately we can't currently link between modules. This is just a placeholder if we ever can in the future, and I would use this link in the above sentence.